commands/join.py

```diff
@@ -0,0 +1,16 @@
+from peewee import *
+from models.users import Users
+
+class BotCommand:
+    def __init__(self, client):
+        self.client = client
+
+    def aliases(self):
+        return ["join"]
+
+    async def handle(self, message, command, arguments):
+        if not Users.select().where((Users.discord_id == message.author.id) & (Users.server_id == message.guild.id)):
```

Comment:
I was going off of the stuff here: docs.peewee-orm.com/en/latest/peewee/query_operators.html

> Although you may be tempted to use python’s in, and, or and not operators in your query expressions, these will not work. The return value of an in expression is always coerced to a boolean value. Similarly, and, or and not all treat their arguments as boolean values and cannot be overloaded.
>>Use & instead of and

Reply:
interesting